<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAircraftModelStatisticsDialog</class>
 <widget class="QDialog" name="CAircraftModelStatisticsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>268</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aircraft model statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tw_ModelStatistics">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_GeneralStatistics">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_GeneralStatistics">
       <item>
        <widget class="QTextEdit" name="te_GeneralStatistics">
         <property name="documentTitle">
          <string>Statistics</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Matrix">
      <attribute name="title">
       <string>Matrix</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_MatitxInfo">
         <property name="text">
          <string>Generate matrix and display in browser</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="pb_GenerateMatrix">
         <property name="text">
          <string>generate matix</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_AircraftModelStatistics">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bb_AircraftModelStatistics</sender>
   <signal>accepted()</signal>
   <receiver>CAircraftModelStatisticsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_AircraftModelStatistics</sender>
   <signal>rejected()</signal>
   <receiver>CAircraftModelStatisticsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bg_Layout"/>
  <buttongroup name="bg_Chart"/>
 </buttongroups>
</ui>
